torch/_export/serde/serialize.py

```diff
@@ -1364,6 +1428,34 @@ def deserialize_outputs(self, serialized_node: Node, fx_node: torch.fx.Node):
         self.deserialize_multiple_outputs(serialized_node, fx_node)
 
     def deserialize_multiple_outputs(self, serialized_node: Node, fx_node: torch.fx.Node) -> None:
+        def generate_getitems(meta_val, fx_node: torch.fx.Node, args: Iterable[Argument]):
+            for idx, arg in enumerate(args):
+                if arg.as_tensor is not None:
+                    name = arg.as_tensor.name
+                    individual_output = self.graph.create_node(
+                        "call_function",
+                        operator.getitem,
+                        (fx_node, idx),
+                        name=name,
+                    )
+                    self.sync_fx_node(name, individual_output)
+                    meta_val.append(self.serialized_name_to_meta[name])
+                    # The derived `getitem` nodes should have the same stacktrace as the
+                    # original `fx_node`
+                    individual_output.meta.update(self.deserialize_metadata(serialized_node.metadata))
```

Comment:
nit... 

`self.deserialize_metadata(serialized_node.metadata)` is returning the same result in line 1445 and 1454? 
This result can be cached in advance.